CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Please format the changes as follows:
 + BugFixes:
 + Updates:
 
+# 1.0.26
```

Comment:
I recently version bumped to 1.0.25 , we don't need to version bump for this change since master has 1.0.24 still

Reply:
This is targeting master, thus master has 1.0.25 before this change. I'm bumping the version number so that we produce packages in higher precedence that what was released as 1.0.25

Reply:
This PR is targeting develop branch, which is should be fine for testing since the only difference between develop and master are the MIT license change and nit bugfixes.

Reply:
Sorry, yes, I meant develop. I sometimes get tripped up because of the similar-yet-different branch name schemes among various repos.